### PR TITLE
Updated our example

### PR DESCRIPTION
**Pull request by glennawatson**

### CODE REVIEW

**Comment by glennawatson** [comment]
I think your expando box thing will be useful in this case @worldbeater 

I've added about 100 lines of text.

**Comment by glennawatson** [comment]
The only reason why it became NuGet, is Octokit you cannot do cancellation, has no CancellationToken support and the code for NuGet was pretty much identical.

**Comment by glennawatson** [comment]
https://github.com/octokit/octokit.net/issues/1840

**Comment by glennawatson** [comment]
http://prismlibrary.github.io/docs/xamarin-forms/Getting-Started.html

Our getting started is sort of close to the Microsoft style, not quite sure I like their breakdown technique but might give you more context.

File: input/docs/getting-started/index.md [hunk 1]
```diff
@@ -21,14 +21,27 @@ Install-Package ReactiveUI.WPF
...
 ### 3. Create a new AppViewModel class
 ```csharp
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Windows;
+using NuGet.Configuration;
+using NuGet.Protocol;
+using NuGet.Protocol.Core.Types;
```
Comment (worldbeater) [inline input/docs/getting-started/index.md]: Is it worth showing up usings here? 
Visual Studio (especially with ReSharper) can auto-import them for us.

**Pull request merged**